src/sentry/static/sentry/app/views/settings/project/filtersAndSampling/rules/rule/index.tsx

```diff
@@ -39,22 +59,37 @@ function Rule({rule, onEditRule, onDeleteRule, disabled}: Props) {
           disabled={disabled}
         />
       </Column>
-    </React.Fragment>
+    </Wrapper>
   );
-}
+});
 
 export default Rule;
 
-const Column = styled('div')`
-  display: flex;
+const Wrapper = styled('div')<WrapperProps>`
+  display: grid;
   align-items: center;
+  :not(:last-child) {
+    border-bottom: 1px solid ${p => p.theme.border};
+  }
+  ${p => layout(p.theme)}
+  ${p =>
+    p.isDragging &&
+    `
+      background: ${p.theme.background};
+      box-shadow: 0 0 15px rgba(0, 0, 0, 0.15);
+      opacity: 0.8;
```

Comment:
I wouldn't make the dragging item translucent. It is the current item that the user is interacting with and should feel solid in my opinion.

Reply:
I will update that 😉